doc/release/release_notes_v1_2.md

```diff
@@ -0,0 +1,328 @@
+
+### After Creating 200 Pools, daos_io_server Crash (signal 11) When Querying the Pools ([DAOS-6505](https://jira.hpdd.intel.com/browse/DAOS-6505))
+
+
+ This will be fixed in a future release.
+
+### Soak: ior -a MPIIO - daos client report DER_INVAL with oclass RP_2GX ([DAOS-6308](https://jira.hpdd.intel.com/browse/DAOS-6308))
+
+If failures reduce the number of available targets below the number required for an object, writes will not be possible until the system is repaired via reintegration, and data will be vulnerable to additional failures until that reintegration is complete. 
+
+ This will be fixed in a future release.
+
+###Add support for co-located (targets) shards in placement ([DAOS-5499](https://jira.hpdd.intel.com/browse/DAOS-5499))
+
+If you have a widely striped object (or even just a "normal object" but a very small cluster) one or more failures might mean that your objects require more targets to place than are available on the system. 
+
+ This will be fixed in a future release.
+
+### fio/libaio not working over dfuse with direct_io ([DAOS-5299](https://jira.hpdd.intel.com/browse/DAOS-5299))
+
+Trying to run fio over dfuse results in EINVAL errors in normal usage.  When fio is using libaio as a backend, libaio is calling io_setup() which works, however, io_submit() is failing with EINVAL.
+
+Currently there is a bug where dfuse doesn't set the O_DIRECT flag. 
+As a workaround, adding a --disable-directio flag to dfuse and setting this for the fio test.
+
+This will be fixed in a future kernel update.
+
+### The dmg storage scan may report incorrect NUMA socket ID
+The ndctl package maintainers have confirmed that this is a kernel regression
+that has been resolved in CentOS 8. Some DAOS nodes may not report the correct
+NUMA socket ID when running a "dmg storage scan." This appears to be a
+regression in the CentOS7.7 kernel rather than an ndctl issue. This is due to
+different versions of ndctl provisioning different JSON namespace details
+(storage scan command reads the "numa_node" field). The regression also affects
+the operation of the "dmg config generate" command which isn't able to detect
+correct NUMA affinity for the PMem namespaces required for the config.
+The regression has been fixed in the CentOS7.9 kernel.
+
+A fix for this is not planned.
+
+## Fixed Issues Details
+### PSM over OPA is not fully functional
+If you must evaluate DAOS on an OPA fabric, we recommend using IP over
+fabric/sockets for evaluation.  For benchmarking, PSM2 over OPA offers a
+better picture of DAOS performance, but the PSM2 provider is not fully
+functional. For a discussion of issues and options, contact us via the
+[mailing list](https://daos.groups.io/g/daos).
+
+### Replication and Self-healing
+While replication and self-healing features have been fully implemented,
+those features haven't been thoroughly tested in 1.2. As a result, they are
+supported as preview features and will be fully supported in the next release.
```

Comment:
Is this left over from 1.0?  I believe we are claiming support for replication & self healing